### PR TITLE
[RS-826] New job to calculate newtab visits -> activity stream sessions

### PR DESCRIPTION
**Pull request by sergiosonline**
Based on: https://console.cloud.google.com/bigquery?sq=687003905036:dc767e16ce8d4aed84282531d6d51f1c 

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1711)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      'GB',
+      'IN',
+      'IT',
+      'JP',
+      'MX',
+      'US'
+      )
+    AND ((page ) IN ('about:home', 'about:newtab')
+    AND (user_prefs ) >= 258)
+    AND `mozfun`.norm.browser_version_info(version).major_version >= 91
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql:25]: You'll want to run
`bqetl format sql/moz-fx-data-shared-prod/ads_derived` locally to format this file

File: sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    AND DATE(submission_timestamp) = @submission_date
+  GROUP BY
+      1, 2
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql:28]: The guidance in the [DTMO SQL style guide](https://docs.telemetry.mozilla.org/concepts/sql_style) is to be explicit with this kind of thing

```suggestion
  GROUP BY
      submission_date,
      country
```

File: sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      'FR',
+      'GB',
+      'IN',
+      'IT',
+      'JP',
+      'MX',
+      'US'
+      )
+    AND ((page ) IN ('about:home', 'about:newtab')
+    AND (user_prefs ) >= 258)
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql:24]: Just some cleanup

```suggestion
    AND page IN ('about:home', 'about:newtab')
    AND user_prefs >= 258
```

File: sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          mozfun.map.get_key(e.extra, "source") = 'about:home'
+          AND nt.metrics.string.newtab_homepage_category = "enabled"
+        )
+        OR
+        (
+          mozfun.map.get_key(e.extra, "source") = 'about:newtab'
+          AND nt.metrics.string.newtab_newtab_category = "enabled"
+        )
+      )
+  GROUP BY 1, 2
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql:69]:
```suggestion
  GROUP BY submission_date, country
```

File: sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  a_s.country,
+  newtab_clients,
+  as_clients,
+  newtab_visits,
+  as_sessions,
+  ROUND(SAFE_DIVIDE(as_sessions, newtab_visits), 2) as conversion_factor
+FROM newtab_visits n_s
+INNER JOIN activity_stream a_s
+  USING (submission_date, country)
+ORDER BY 2, 1
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/ads_derived/nt_visits_to_sessions_conversion_factors_daily_v1/query.sql:83]:
```suggestion
ORDER BY country, submission_date
```

**Review by curtismorales — COMMENTED** [review]
This looks good to me -- did you make a decision about the DAG? should we just make `bqetl_ads`?

**Comment by sergiosonline** [comment]
@curtismorales : I implemented your suggestions. I'm still unable to run the `bqetl` command, but I'll work to configure it in the next few weeks. Can you update the DAG, run `bqetl format...`, and let me know if you have any more feedback?

**Review by curtismorales — APPROVED** [review]
I think this is good to go. @sergiosonline take a quick look and ping me when you want me to merge it

**Pull request merged**